lib/main.js

```diff
@@ -70,6 +71,11 @@ ava.test.before(async () => {
   console.log(`Downloading device type OS into ${imagePath}`)
   await resinio.downloadDeviceTypeOS(options.deviceType, options.resinOSVersion, imagePath)
 
+  if (options.delta) {
+    console.log('Enabling deltas')
```

Comment:
@jviotti no need to log this if it's not enabled right?

Reply:
Looks good!